### DISCUSSION TITLE
Livolo devices adding to Zigbee2mqtt

### DISCUSSION THREAD

**Discussion by Rincovanderbeek** [Koenkk/zigbee2mqtt#5759] [Devices]
Hi,
I know there has been some questions and research for adding new livolo devices to mqtt / HA.
Recently I started with HA and do have several new Livolo switched, sockets etc.  
Hopefully you are willing (and able) to add these to Zigbee2mqtt so it's is possible to use them in HA.
(and explain in a simple way how to use, I am new to this :-))
I noticed you live in NL so if you need some of these devices, I probably am able to loan you if that helps.

**Comment by Koenkk** [comment]
Please check: https://github.com/Koenkk/zigbee2mqtt/issues/5516#issuecomment-753620497 . Because these devices deviate so much from the Zigbee spec I'm currently not interested in adding them myself (because it will like take a lot of time).

**Comment by Nosstrom** [comment]
Hello i have an issue where all my zeigbee livolo devices are not recognisable by zigbee2mqtt i am 6 hours now and cant find a way to make them work...i am on 1.19  PLEASE HELP

**Reply by avlisiur78** [reply]
Hi, 

I also have several Livolo devices, but with a little persistent you can get it to work.

I manage by following the instructions, so I'm working my zigbee on channel 26, the pan_id must be 6756 and you have to have the "ext_pan_id_string: 33,117,141,25,0,75,18,0" set.

After this start pairing, to pair the sockets, first they are recognized as switch, but if you reconfigure (with "permit to join" on) they change to socket, and if you can't make them respond, then, with "permit to join" on, press the pair botton on the socket (light will blink), and keep trying to on and off the swift on the zigbee interface, and refeshing the end point state on the "exposure" option of the device on the zigbee2mqtt, you will notice that will start working.

The issue that I have know, is that if the Livolo socket is paired with more than a router o to the router and the coordinator, I don't know why but it's stops working and give timeout, but if changes to just one of the two, starts to work again.

My network is based on CC2531 (one coordinator with the sourcing version) and 2xCC2531 routers.

**Comment by TychosNovae** [comment]
Hello,
I'm new in HA (few days). My system is with Sonoff Zigbee usb dongle. I have add successfully (pairing) my Livolo TI0001-switch  after change the configuration with channel 26, then restore it to channel 11. Livolo works anyway, so that I think that channel 26 is required only for pairing.

**Reply by btb42** [reply]
how did you achieve this ?  when i try to switch channel, Z2M wants delete coordinator_backup and repair all devices. when i did it, Livolo is not present. if i edit files and add it manualy,  switches did not work.
